.github/workflows/gradlepublishalpha.yml

```diff
@@ -23,6 +23,7 @@ jobs:
       - name: Grant execute permission for gradlew
         run: chmod +x gradlew
       - name: Run publish plugin
+        if: contains(github.event.pull_request.labels.*.name, 'deploy-alpha')
```

Comment:
👍

Reply:
Hope it gonna work 🙏

Reply:
Looks correct according to this: https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#example-using-an-array

Reply:
I'm not sure whether PR is accessible in the push event.

Reply:
Let's see